pkg/ui/workspaces/cluster-ui/src/store/nodes/nodes.fixtures.ts

```diff
@@ -888,6 +888,7 @@ export const getNodeStatus = () => {
           "replicas.leaders_not_leaseholders": 0,
           "replicas.leaseholders": 77,
           "replicas.quiescent": 77,
+          "replicas.uninitialized": 0,
```

Comment:
Is this diff still relevant given we're not adding it to the cluster UI? (anything in addition to the advanced debug page i.e.)